internal/utils/defaults.go

```diff
@@ -0,0 +1,9 @@
+package utils
+
+const (
+	PodSecurityAdmissionEnforcement = "baseline"
+	PodSecurityAdmissionWarning     = "restricted"
+	PodSecurityAdmissionAudit       = "restricted"
+
+	PodSecurityPrivileged = "privileged"
```

Comment:
nit: this is a constant, not a default, unlike the first 3. 
(The first 3 define what we want as default, picked for the ns)